dbtype/bigint.go

```diff
@@ -181,7 +192,57 @@ func (b *BigInt) Scan(src interface{}) error {
 		i = i.Mul(i, big.NewInt(1).Exp(big.NewInt(10), exp, nil))
 	}
 
-	*b = BigInt(*i)
+	b.BigInt = *i
+	b.Valid = true
 
 	return nil
 }
+
+// MarshalJSON implements json.Marshaler
+func (b NullBigInt) MarshalJSON() ([]byte, error) {
+	return b.MarshalText()
+}
+
+// UnmarshalJSON implements json.Unmarshaler
+func (b *NullBigInt) UnmarshalJSON(text []byte) error {
+	if string(text) == "null" {
+		return nil
+	}
+	return b.UnmarshalText(text)
+}
+
+// func (src *Point) AssignTo(dst interface{}) error {
```

Comment:
can you remove some of my old test code here.. and DecodeText and Set and Get and AssignTo methods below please?